The linter fails with:
`pkg/files/sources.go:91:1: exported: exported function NewHTTPSource should have comment or be unexported (revive)
func NewHTTPSource(path string) HTTPSource { return HTTPSource{path, &http.Client{}} }`
but this function already existed. Should I add a comment?

--- a/pkg/files/sources.go
+++ b/pkg/files/sources.go
@@ -88,6 +88,7 @@ type HTTPSource struct {
 	Client *http.Client
 }
 
+// NewHTTPSource returns a new source of type HTTP
 func NewHTTPSource(path string) HTTPSource { return HTTPSource{path, &http.Client{}} }
 
 func (s HTTPSource) Description() string {

